client/gutenberg/editor/init.js

```diff
@@ -24,6 +25,45 @@ const WPCOM_UNSUPPORTED_CORE_BLOCKS = [
 	'core/file', // see D19851 for more details.
 ];
 
+/* eslint-disable prefer-const */
```

Comment:
This would disable `prefer-const` rule for the whole file; how about disabling just "next-line"?
```js
// eslint-disable-next-line prefer-const
```